firmware/controllers/sensors/sensor_type.h

```diff
@@ -77,6 +77,9 @@ enum class SensorType : unsigned char {
 
 	TurbochargerSpeed = 35,
 
+	MapFast = 36,
```

Comment:
code comment with a quick summary of what is fast MAP vs what is slow MAP

Reply:
I guess - they're just averaged (synchronous) map vs asynchronous map